service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1698,7 +1717,20 @@ components:
         description:
           description: Optional. New description to give to this resource. The resource description will not be updated if this is omitted.
           type: string
-
+    UpdateControlledGcpGcsBucketRequestBody:
+      type: object
+      description: Update a GCS bucket's metadata and lifecycle rules.
+      properties:
+        name:
+          description: Optional. New name to give to this resource. The resource name will not be updated if this is omitted.
+          type: string
+        description:
+          description: Optional. New description to give to this resource. The resource description will not be updated if this is omitted.
+          type: string
+        updateParameters:
+          $ref: '#/components/schemas/GcpGcsBucketUpdateParameters'
```

Comment:
Is this optional? Are the sub-parts optional? For example, can I only change the default storage class?